client/layout/masterbar/masterbar-cart/masterbar-cart-button.tsx

```diff
@@ -38,7 +38,9 @@ export function MasterbarCartButton( {
 	useEffect( () => {
 		if ( shouldShowCart ) {
 			reduxDispatch( recordTracksEvent( 'calypso_masterbar_cart_shown' ) );
-		}
+		} else {
```

Comment:
You can't see it in this diff, but there's some indentation errors here. 
<img width="641" alt="Screen Shot 2022-08-18 at 6 02 21 PM" src="https://user-images.githubusercontent.com/2036909/185503025-7eea9528-c55d-4ebb-a15d-f67e2ab22ba6.png">